content/docs/quickstart/objective-c.md

```diff
@@ -9,43 +9,42 @@ description: This guide gets you started with gRPC on the iOS platform in Object
 
 ### Before you begin
 
-#### System requirement
-The minimum deployment iOS version for gRPC is 7.0.
+#### System requirements
 
-OS X El Capitan (version 10.11) or above is required to build and run this
-Quickstart.
+- macOS version 10.11 (El Capitan) or higher
+- iOS version 7.0 or higher
 
 #### Prerequisites
 
-* `CocoaPods`: version 1.0 or higher
+- CocoaPods version 1.0 or higher
 
-   * Check status and version of CocoaPods on your system with command `pod
-     --version`.
-   * If CocoaPods is not installed, follow the install instructions on CocoaPods
-     [website](https://cocoapods.org).
+  Check the status and version of CocoaPods on your system:
 
-* `Xcode`: version 7.2 or higher
+  ```sh
+  $ pod --version
+  ```
 
-   * Check your Xcode version by running Xcode from Lauchpad, then select
-     "Xcode->About Xcode" in the menu.
-   * Make sure the command line developer tools are installed:
-   ```sh
-   [sudo] xcode-select --install
-   ```
+  If CocoaPods is not installed, follow the [CocoaPods install
+  instructions](https://cocoapods.org).
 
-* `Homebrew`
-   * Check status and version of Homebrew on your system with command `brew
-     --version`.
-   * If Homebrew is not installed, install with:
-   ```sh
-   /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-   ```
+- Xcode version 7.2 or higher
 
-* `autoconf`, `automake`, `libtool`, `pkg-config`
-   * Install with Homebrew
-   ```sh
-   brew install autoconf automake libtool pkg-config
-   ```
+  Check your Xcode version by running Xcode from Lauchpad, then select
+  **Xcode > About Xcode** in the menu.
+
+  Make sure the command line developer tools are installed:
+
+  ```sh
+  $ xcode-select --install
+  ```
+
+- [Homebrew](https://brew.sh/)
+
+- `autoconf`, `automake`, `libtool`, `pkg-config`
+
+  ```sh
+  $ brew install autoconf automake libtool pkg-config
+  ```
```

Comment:
@lucperkins: as you can see from this change, an indentation of 2 spaces should be enough.

Reply:
Weird, I guess four spaces is still necessary for ordered lists 🤷‍♂

Reply:
Yes, two for unordered lists, and 4 for (properly formatted) ordered lists :).